File: 3870-count-commas-in-range/3870-count-commas-in-range.java
class Solution {
    public int countCommas(int n) {
        int count=0;
        for (int i=1;i<=n;i++){
            String digit=String.valueOf(i);
            int res= digit.length();
            if (res>=4){
                count+=(res-1)/3;
            }
        }
        return count;
        
    }
}

File: 3870-count-commas-in-range/3870-count-commas-in-range.py
class Solution(object):
    def countCommas(self, n):
        """
        :type n: int
        :rtype: int
        """
        count=0
        for i in range(1,n+1):
            digit=len(str(i))
            if digit>=4:
                count+=(digit-1)//3 # Imp formula ---> d=4  d-1//3=1 (1000)--> 1 comma
        return count
              
        